camel/model_backend.py

```diff
@@ -39,14 +43,30 @@ class OpenAIModel(ModelBackend):
     r"""OpenAI API in a unified ModelBackend interface."""
 
     def __init__(self, model_type: ModelType, model_config_dict: Dict) -> None:
```

Comment:
```suggestion
    def __init__(self, model_type: ModelType, model_config_dict: Dict[str, Any]) -> None:
```

Reply:
It would be better if we set model_type and model_config_dict as keyword arguments to follow like `OpenAIModel()` to return a `ModelType.GPT_3_5_TURBO`.

Reply:
Left mandatory